samples/browser/src/main/java/org/mozilla/samples/browser/downloads/DownloadService.kt

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.samples.browser.downloads
+
+import mozilla.components.feature.downloads.AbstractFetchDownloadService
+import mozilla.components.lib.fetch.httpurlconnection.HttpURLConnectionClient
+
+class DownloadService : AbstractFetchDownloadService() {
+    override val httpClient = HttpURLConnectionClient()
```

Comment:
Can we get this from the context/components?

Reply:
I'm not quite sure if we want that every app have to create their own service to use this functionality, Should AC provide a default and avoid extra steps for the consumers?

Reply:
I think it's okay to force the app to provide the dependency here. The app team should make a conscious decision. The only possible default is `HttpURLConnectionClient` and that's the opposite of what we'd recommend for Mozilla apps.